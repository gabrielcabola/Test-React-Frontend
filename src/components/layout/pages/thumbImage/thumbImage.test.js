import React from "react";
import renderer from 'react-test-renderer';
import ThumbImage from './';

it('renders correctly', () => {
  const tree = renderer
    .create(<ThumbImage />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});