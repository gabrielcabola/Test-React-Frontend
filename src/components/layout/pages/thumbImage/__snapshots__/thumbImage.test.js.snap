// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="thumbImage"
>
  <img
    src="https://thumbs.static-thomann.de/thumb/thumb120x/pics/prod//undefined"
  />
</div>
`;
